src/index.d.ts

```diff
@@ -65,12 +65,12 @@ export function getCookieConsentValue(name?: string): string;
  * Remove the cookie on browser in order to allow user to change their consent
  * @param {*} name optional name of the cookie
  */
-export function resetCookieConsentValue(name?: string);
+export function resetCookieConsentValue(name?: string): string;
```

Comment:
Please change this to "void" as the resetCookieConsentValue function doesn't return anything.

Reply:
@Mastermindzh Done, thanks for follow up